examples/powertools-examples-core/gradle/README.md

```diff
@@ -0,0 +1,34 @@
+#  Powertools for AWS Lambda (Java) - Core Utilities Example with Gradle
+
+This project demonstrates the Lambda for Powertools Java module deployed using [Serverless Application Model](https://aws.amazon.com/serverless/sam/) with
+[Gradle](https://gradle.org/) running the build. This example is configured for Java 1.8 only; in order to use a newer version, check out the Gradle 
+configuration guide [in the main project README](../../../README.md).
```

Comment:
should we mention the fact that they can use the `sam init` (with proper choices) and create this automatically?

Reply:
Done!